### PR TITLE
Add ShowPicker function to date, time, and file inputs

### LINKED ISSUE #3374: Show picker manually by using new showPicker method
https://developer.chrome.com/blog/show-picker/

### PR DESCRIPTION
**Pull request by stsrki**
Closes #3374

### CODE REVIEW

File: Source/Blazorise/Components/FileEdit/FileEdit.razor.cs [hunk 1]
```diff
@@ -253,6 +253,15 @@ public ValueTask RemoveFile( int fileId )
             return JSFileEditModule.RemoveFile( ElementRef, ElementId, fileId );
         }
 
+        /// <summary>
+        /// Show a browser picker for file input.
+        /// </summary>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public Task ShowPicker()
+        {
+            return JSUtilitiesModule.ShowPicker( ElementRef, ElementId ).AsTask();
```
Comment (David-Moreira) [inline Source/Blazorise/Components/FileEdit/FileEdit.razor.cs:262]: Isn't this the same as the regular existing `Open` for the `FileEdit` ?

File: Documentation/Blazorise.Docs/Pages/Docs/Components/Dates/DateEditPage.razor [hunk 2]
```diff
@@ -64,6 +64,25 @@
     <DocsPageSectionSource Code="DateEditDateTimeExample" />
 </DocsPageSection>
 
+<DocsPageSubtitle>
+    Functions
```
Comment (David-Moreira) [inline Documentation/Blazorise.Docs/Pages/Docs/Components/Dates/DateEditPage.razor:83]: I think we've used the name `Methods` on other places, `Datagrid`, `Autocomplete`, I honestly have no real preference. But I think in C# the naming `Method` is used more?  We should choose one or the other.

**Review by David-Moreira — APPROVED** [review]
Looking good. Would we ever like to call `ClosePicker`?

File: Source/Blazorise/Components/FileEdit/FileEdit.razor.cs (hunk 1 above)
Comment (stsrki) [reply Source/Blazorise/Components/FileEdit/FileEdit.razor.cs:262]: We don't have `Open` on FileEdit.

File: Documentation/Blazorise.Docs/Pages/Docs/Components/Dates/DateEditPage.razor (hunk 2 above)
Comment (stsrki) [reply Documentation/Blazorise.Docs/Pages/Docs/Components/Dates/DateEditPage.razor:83]: Agree. Methods make more sense.

File: Source/Blazorise/Components/FileEdit/FileEdit.razor.cs (hunk 1 above)
Comment (David-Moreira) [reply Source/Blazorise/Components/FileEdit/FileEdit.razor.cs:262]: It is not directly exposed. It's in the module. We had a user complaining he'd access the module and couldn't call `Open` before. With this, I guess we have two ways to open it?
https://github.com/Megabit/Blazorise/pull/4204

Comment (stsrki) [reply Source/Blazorise/Components/FileEdit/FileEdit.razor.cs:262]: Oh, that. Then the JSmodule version doesn't make much sense anymore.

**Comment by David-Moreira** [comment]
> Looking good. Would we ever like to call `ClosePicker`?

Did you notice this? You didn't comment on this.

**Comment by stsrki** [comment]
> > Looking good. Would we ever like to call `ClosePicker`?
> 
> Did you notice this? You didn't comment on this.

Yeah, I noticed it, but I was driving 😅.

There is no ClosePicker API, so it's impossible to make it.

image

**Pull request merged**